Zeeguu-API-iOS-debug.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name         = "Zeeguu-API-iOS-debug"
+  s.version      = "1.0.1"
+  s.summary      = "An API for Zeeguu for iOS"
+  s.description  = <<-DESC
+  				   The iOS Framework that acts as a layer in front of the Zeeguu server.
+                   DESC
+  s.homepage     = "https://github.com/JorritO/Zeeguu-API-iOS"
+  s.license      = "MIT"
+  s.author    = "Mircea Filip Lungu"
```

Comment:
You're the author Jorrit :) I'm the producer, editor, or something else but you're the author!